### PR TITLE
Added emoji from stickers implementation

### PR DESCRIPTION
**Pull request by unauthorized-401**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -2,24 +2,20 @@
...
 
-const __DEV__ = process.env.NODE_ENV === 'dev';
+const botsList = Object.keys(bots).map(botName => bots[botName]);
 
 // global for debugging
-const jabber = global.jabber = require('./bots/jabber');
-const telegram = global.telegram = require('./bots/telegram');
+Object.keys(bots).forEach(k => {
+  global[k] = bots[k];
+});
```
Comment (ForNeVeR) [inline app.js]: Do we really need to store bots in global variables? I can't see where are they used.

File: bots/jabber/network.js [hunk 2]
```diff
@@ -0,0 +1,2 @@
+const xmpp = Symbol.for('XMPP');
+module.exports = xmpp;
```
Comment (ForNeVeR) [inline bots/jabber/network.js]: Just an idea: let's place all the networks in a single file? I can't see a reason to keep a bunch of files 2 lines long.

File: bots/telegram/client.js [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+      return msg.username;
+    }
+
+    if (msg.last_name) {
+      return `${msg.first_name} ${msg.last_name}`;
+    }
+
+    return msg.first_name;
+  }
+}
```
Comment (ForNeVeR) [inline bots/telegram/client.js]: Honestly that looks strange. I can't see why you need a `class` for that single static function.

File: app.js (hunk 1 above)
Comment (unauthorized-401) [reply app.js]: Already discussed and deleted those globals

File: bots/jabber/network.js (hunk 2 above)
Comment (unauthorized-401) [reply bots/jabber/network.js]: The idea was that outside world doesn't need to know anything about the kind of network, this is needed just to skip double handling of messages sent through bus.

File: bots/telegram/client.js (hunk 3 above)
Comment (unauthorized-401) [reply bots/telegram/client.js]: This is done this way for having common interface for all *Message classes, code with similar logic is static and named as test is a part of other messages types. These kinds of methods are for checking if Telegram message could be transformed to our class, we can rename them all or re-structure as non-statics or whatever but in such case this change should be applied to all methods of this kind

File: bots/jabber/network.js (hunk 2 above)
Comment (ForNeVeR) [reply bots/jabber/network.js]: Eh, okay then.

File: bots/telegram/client.js (hunk 3 above)
Comment (ForNeVeR) [reply bots/telegram/client.js]: Ok.

**Comment by ForNeVeR** [comment]
Thanks for your contribution, everything is okay. I'm merging that.

**Pull request merged**